setup.py

```diff
@@ -54,6 +54,7 @@
         "Flask-Celery-Helper",
         "Flask-Migrate",
         "Flask-OAuthlib",
+        "Flask-Recaptcha",
```

Comment:
Ah right, forgot that we also needed it in `requirements.txt`. Should be there now.